kernel-ewasm/tests/utils/tester.ts

```diff
@@ -0,0 +1,160 @@
+// This module holds the Tester class, which contains most of the boilerplate
+// required in tests.
+import { Contract } from "web3-eth-contract";
+import * as utils from 'web3-utils';
+import { newKernelInstance, web3, createAccount, KernelInstance, deployContract, normalize, EntryCap, WriteCap, RegisterCap, NewCap} from '../utils'
+
+const assert = require('assert')
+
+const BN = require('bn.js')
+const Web3 = require('web3');
+const fs = require("fs");
+const path = require("path")
+const http = require('http')
+
+// The tester class is an object for executing tests on the kernel.
+export class Tester {
+    entry_proc: TestContract;
```

Comment:
I believe you can just do:
```suggestion
    entry_proc: TestContract = null;
```